gst-libs/gst/androidjni/gstjniutils.h

```diff
@@ -79,4 +79,93 @@ _type gst_jni_get_##_name##_field (JNIEnv *env, jobject obj, jfieldID fieldID);
 
 DEF_GET_TYPE_FIELD (gint, int, Int, G_MININT)
 DEF_GET_TYPE_FIELD (glong, long, Long, G_MINLONG)
-#endif
+
+
+/* Macros have next rules:
+   J_CALL_<TYPE> (), J_CALL_STATIC_<TYPE> () - first parameter is a variable
+   to write to, if it's not J_CALL_VOID () or J_CALL_STATIC_VOID ().
+   If exception occured, it jumps to "error" label, and the variable
+   is kept untouched.
```

Comment:
I have the way-around opinion: this pattern of having an "error" label in each function should be used in all the code we write, because it simplifies the code, and at the same way forces the code to check and log every unexpected case.